library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt

```diff
@@ -233,19 +233,19 @@ class GroupTest {
         runBlocking {
             boClient.conversations.newGroup(
                 listOf(
-                    alix.walletAddress,
-                    caro.walletAddress
+                    alixClient.inboxId,
+                    caroClient.inboxId
                 )
             )
         }
         runBlocking { alixClient.conversations.syncGroups() }
         val group = runBlocking { alixClient.conversations.listGroups().first() }
-        runBlocking { group.removeMembers(listOf(caro.walletAddress)) }
+        runBlocking { group.removeMembers(listOf(caroClient.inboxId)) }
```

Comment:
`removeMembers` still takes addresses. We just resolve them to `inbox_id` for you under the hood.

We do have a `remove_members_by_inbox_id` and `add_members_by_inbox_id` method, although now that I think about it we don't expose those via FFI. That's an easy add.

I'd imagine that'd be useful for tests, and probably some actual user flows too.